samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
6fDscifYZsjvhf1fY8CvEIrFepnDblFa3Sws34Z9VhmHkuyNw5hfJqaulj/wAOR7RyB0/hOisYtp6gcTG/8zf/AJIVlauD5TRRloLF8rTPpuVRvtzC+fDa2XviiP8AGP6rR+1c/cyFvo8/6l1+pq8/0c/pLvH9nvJHjn7Kura1jBd7msH4iBfyHevEVGPVD+MpaOTAGfMa/NVz3Em5JJPEkkk+pXEtYl8q+5JD0+T+d/Y9HiO0Y1ELb/jfoPRvH3svPTTOeS57i4nvP96LRFUsunZ8zNCqmFa7UERFEShERAEREAREQBZCwshAYREQBERAFb4Ls1VVbHPpohI1jsjiXxss6wNusRfQj3VQvtvRjR7vDYiRYzvlnPjd2Vp/hY1AfLsW2SrKaIzTwhkbS1pcJInWLjYaNcTxK5MFwSoq3OZTR7xzGhz+sxgaCbDVxA9PAr630oD/AIXP4Ppz/wC5iz0b4L9momucLS1Npn34hpH3bfRutubigPmlXsRXxRvlkga2OJjpHu3sBysaLk6O5BYg2Jr3xtmbTjdvYJQ4yQN6hGYEguuNF9R6Q5T9hfC3t1csFIzxMsgB/lDl5raOixSlo5Xy4nE6FjGxGNkbGlzXkRhoOTTtc0B8yBRes2f6P6qpYJXFlPE4XY6UEueO4hg7vEkKyruiyoYwuiqYpnDXdljoS7wBLnC/nbzQHgUXoNmNk5q187GubCaYtbJvQ7tuLhl07xkN1cM6M6neSB00LIowCZjn63VDjlbxIF9SSNQgPDovZ4D0c1NQwSSPbSxuALA9pfI5p4Ex3GW45m/gu/EeiuZrS6CpZM4C+7ewwl3k7MRfzt5oD56itsD2dqKqd1PEzK+O++Ml2thscpzd973Fhrp4Fezj6KwLB9e0Pdwa2IfIGS5QHzZF6favYqahaJTIyaEuDDI0Fha43tmYSbA243P0vtszsLU1jBLdsEDuzJIC4vHNjBa48SR4XQHlkX0eq6KJA0mKsY54GjZIjGHH8wc63sVS7P7BT1D5myObB9mmbDM1+YuOgccpGnZIIP4ggKrD9l62dgkhpJHxu1a8mOMOHMZ3C48QqqpjMb3xvGV8bnMe24OV7SWuFwbaEEaL9DYtTTOp3R0kjIJcrWxvc3M2NoIvZtuOUED0XJsjgzqSn3Uj2ySvmlnlkaCA973Xvr32DfZAfAUXodscFfT1b2yyxvknz1TizMA3eSP0ObvuD8ua8+QgMIiIAshYWQgMIiIAiIgFjwAuToBzPcF+iqKFtNTRsOjKaBrSfwxs1PyK+GbIUm+r6SM8DOx7vyx/eH5Mt6r630i14jw2psQXStbABfiJCGu/lzn0QHW6lFfQQCa1qiOjqJABo6xjlc3yNi3yK624gHVTqdtvuYGyzfhMjrRN9mSH+FcGA17IcKpJpHWbFh9M9/PqwtuAOfcqvo1ldLFV1sthJWVTnceEbGhrW+QOYDwCA69ofvcRwun4iN1RXPH/AG2ZGfzP+S69o6Zs8tHSvsWOmdUysP7cUDb5T4bySK/hdceGPEmLV0pIy00FNRsJI7TrzSW92qo2i2lZT41TZyNzHTGKRw1yGd2Yu05ZIifC6A7Ok/aB9NTxxQvLJakvGdujmRMAzFp7iS5ov4m2q8Xs50gz0sJhdH9p67ntfLK/M1pA6nA3FwTx/aX0TajZeHEmwP3zm7vMWSRZXh0b8txyPZaQVV7QS4Zh1OGfZqWWdjMsUTmRySPcBo6R1rgd5cfTkgO7o+l30NRWmNsZral8mRpLgAxrYuPf1mPP7yhwzal9ViclLCGClp2SmR5GZ0z2kMsDwa3M7lrl46rvw+UUeFtc4szwUhmeG5QN9kMjgAOHXJ0Xyzo+xttJWNfM60UsboZHnXLmLXB58LtF/AkoD3fSpjz4IIoIXlj6kvzPbcObCy1wD3ElwF+QKi6IZZDSVGdzjG2ezMxJDeoC+xPAag+ZKuNqdlYcSEEm+czdh2SSLLI18b7Ejl3Cx8+K4NoHYbQUbaaSOOfdtO6pnHO6R51L3jgLnUuI8u4IDmwnaOkhxXEWOljaypdTOZPcGMyMjAe0v4DU8eF83erfHNj4KueOsE0sU8e7ySxFj2nIczDZwI0J7rXXDsph+GVlPG9tLSGXI0TxhjQ5ktrO6vEC9yDyWNk9g3UNSZhWPfEGvAiDMmcHQbw5rG3HQDUA6cEBw43hde+ppaOrqhU0VXN1i2KKI/dAzFjgBcXDDYg668FedIGNOo6ImHqSSubTxEAfd3BJcB3Wa028SFTbYbaQx1dE2JwlbTTmWodGcwaDG+IsBGjnBsjzbmAPL0GPYRBilMwMn6geJYporPAdYggjycdNCgPH9D00rpavM97o8kbnZnOd96XGx17yA6577BegrA52OwMZLK1jaM1E0TXvDHkF8bS5oNies3j8I5LqpmUeD0pa6S1yXuzWMs8lu5o8gOQ7+8qn6O6x1VU19fLZpkMUMbb9hguco8hu/M3KA16W8VliZSwwSSxySPkkvE5zCQ0BoaS0gkEycPwr0G1FQ6lwuY7x+8jpmwNkzOL964NiDs3HNd17+q8NtxiEbsaphI8CGmdSNkOlm9cSuJPKzm38AvfbUYfDWUro31GSO7ZTJGWOuGdbv0I/QID4RUVUkjs8kkkj7AZ5HOe6w4C5N7KJAUQBERAFkLCyEBhERAEREBgi+hF/NYbGBwAHkAFsiA13YvewvzsEdGDxaD5gFbIgNTGOQ04aBZAtwFvJZRAbxyuaCGvc0HiGktB9lpb9fVEQGojHHKL+QWyIgJIp3tBDHvYDxDXOaD52Kj/rqfEoiAy02II0I4EaEeRUstXI4ZXyyvb8LnvcPYlQogC3hlcwkse5hPEsJaT7LREBlziSSSSTxJ1J8ytXMB4gHzF1lEBgC3DRY3Y45RfnYLZEAREQBERAFkLCyEBhERAEREAREQBERAEREAREQBERAEREAREQBERAEREAREQBERAEREAWQiID/9k=';
```

Comment:
Looks like you made this request while I was doing the others and I didn't see it. I have replaced the image with a homemade abstract image that doesn't require permissions and won't need to be updated if MS changes its icons again in the future.